src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/InputTestInvalidAtSeeReference.java

```diff
@@ -1,7 +1,7 @@
 package com.puppycrawl.tools.checkstyle.checks.javadoc;
 
 /**
- * @see javax.swing.tree.DefaultTreeCellRenderer.getTreeCellRendererComponent()
+ * @see "javax.swing.tree.DefaultTreeCellRenderer.getTreeCellRendererComponent()"error
```

Comment:
The UTs mentioned in [this](https://github.com/checkstyle/checkstyle/pull/4550#issuecomment-312529807) comment continue to fail because of this change.